website/pages/docs/release-notes/1.5.0.mdx

```diff
@@ -0,0 +1,64 @@
+---
+layout: docs
+page_title: 1.5.0
+sidebar_title: 1.5.0
+description: |-
+  This page contains release notes for Vault 1.5.0.
+---
+
+# Vault 1.5.0
+
+## Vault 1.5 Release Highlights
```

Comment:
Is there a particular sorting order for this list?

Reply:
No - i think this is just what Product wrote down in the draft.

Reply:
I don't know - a google doc went around for comments over the course of the last week but probably could have been more circulated.